### ISSUE TITLE
Foreman does not terminate child precesses

### ISSUE THREAD

**Issue by szyablitsky** [ddollar/foreman#618]
Updated foreman from 0.78.0 to 0.80.2 and now after hitting Ctrl-C foreman exits but does not terminate child processes. I have to terminate all of them manually. Reverting back to 0.78.0 solves this problem. Ubuntu 16.04

```
^C10:57:12 system   | SIGINT received, starting shutdown
10:57:13 system   | sending SIGTERM to all processes
10:57:13 rails.1  | terminated by SIGTERM
10:57:13 hot.1    | terminated by SIGTERM
10:57:13 client.1 | terminated by SIGTERM
10:57:13 server.1 | terminated by SIGTERM
```

```
$ ps -f
UID        PID  PPID  C STIME TTY          TIME CMD
sergey    1980  1974  0 10:47 pts/17   00:00:00 zsh
sergey    3786  1338  6 10:56 pts/17   00:00:04 puma 3.4.0 (tcp://0.0.0.0:3000) [ins_web]
sergey    3788  1338  0 10:56 pts/17   00:00:00 sh -c rm app/assets/webpack/* || true && HOT_RAILS_PORT=3500 npm run hot-assets
sergey    3790  1338  0 10:56 pts/17   00:00:00 sh -c npm run build:dev:client
sergey    3793  3790  0 10:56 pts/17   00:00:00 npm
sergey    3794  1338  0 10:56 pts/17   00:00:00 sh -c npm run build:dev:server
sergey    3795  3794  0 10:56 pts/17   00:00:00 npm
sergey    3798  3788  0 10:56 pts/17   00:00:00 npm
sergey    3829  3793  0 10:56 pts/17   00:00:00 sh -c (cd client && npm run build:dev:client --silent)
sergey    3830  3829  0 10:56 pts/17   00:00:00 npm
sergey    3837  3798  0 10:56 pts/17   00:00:00 sh -c (cd client && npm run hot-assets)
sergey    3838  3837  0 10:56 pts/17   00:00:00 npm
sergey    3845  3795  0 10:56 pts/17   00:00:00 sh -c (cd client && npm run build:dev:server --silent)
sergey    3846  3845  0 10:56 pts/17   00:00:00 npm
sergey    3865  3838  0 10:56 pts/17   00:00:00 sh -c babel-node server-rails-hot.js
sergey    3866  3865  0 10:56 pts/17   00:00:00 node /home/sergey/projects/ins_web/client/node_modules/.bin/babel-node server-rails-hot.js
sergey    3871  3846  0 10:56 pts/17   00:00:00 sh -c webpack -w --config webpack.server.rails.build.config.js
sergey    3872  3871 18 10:56 pts/17   00:00:13 node /home/sergey/projects/ins_web/client/node_modules/.bin/webpack -w --config webpack.server.rails.build.config.js
sergey    3877  3830  0 10:56 pts/17   00:00:00 sh -c webpack -w --config webpack.client.rails.build.config.js
sergey    3878  3877 25 10:56 pts/17   00:00:17 node /home/sergey/projects/ins_web/client/node_modules/.bin/webpack -w --config webpack.client.rails.build.config.js
sergey    3883  3866 28 10:56 pts/17   00:00:19 /home/sergey/.nvm/versions/node/v4.4.3/bin/node /home/sergey/projects/ins_web/client/node_modules/babel-cli/lib/_babel-node server-rails-hot.js
sergey    4422  1980  0 10:57 pts/17   00:00:00 ps -f
```

**Comment by kossnocorp** [comment]
Also have this problem.

**Comment by ddollar** [comment]
Should be fixed by reverting 8c003b6d67ce34551741dcd50947b69def517064

Releasing as 0.81.0 shortly
